packages/yew/src/html/component/scope.rs

```diff
@@ -498,6 +511,72 @@ mod feat_io {
     }
 }
 
+#[cfg_attr(documenting, doc(cfg(feature = "hydration")))]
+#[cfg(feature = "hydration")]
+mod feat_hydration {
+    use super::*;
+
+    use crate::virtual_dom::Fragment;
+
+    impl<COMP: BaseComponent> Scope<COMP> {
+        /// Hydrates the component.
+        ///
+        /// Returns a pending NodeRef of the next sibling.
+        ///
+        /// # Note
+        ///
+        /// This method is expected to collect all the elements belongs to the current component
+        /// immediately.
+        ///
+        /// We don't remove the comment node at the moment as it's needed to maintain the
+        /// structure.
+        pub(crate) fn hydrate_in_place(
+            &self,
+            parent: Element,
+            fragment: &mut Fragment,
+            node_ref: NodeRef,
+            props: Rc<COMP::Properties>,
+        ) {
+            // This is very helpful to see which component is failing during hydration
+            // which means this component may not having a stable layout / differs between
+            // client-side and server-side.
+            #[cfg(debug_assertions)]
+            log::trace!(
```

Comment:
How can `gloo-console` turn off trace level logging (with debug build)? This is very easy to do with `log` crate.

I find this hydration message is very verbose when hydrating big applications and wish it to remain off unless I am having issues with hydration.

Reply:
Right, it can't disable those with a flag. I didn't know that log was used to avoid printing messages, even on debug mode, unless trace is enabled.

I assume they're being printed to the browser console so it's not an issue since the messages there could be filtered by level